ote/src/clj/ote/services/login.clj

```diff
@@ -45,27 +52,33 @@
           (encrypt/passlib->buddy (:password login-info)))
       (if (:email-confirmed? login-info)                    ;;If email not confirmed send proper error
         ;; User was found and password is correct, return user info
-        (with-auth-tkt
-          (http/transit-response
-            {:success? true
-             :session-data
-             (let [user (users/find-user db (:id login-info))]
-               (transport-operator/get-user-transport-operators-with-services db (:groups user) (:user user)))}
-            200)
-          (cookie/unparse "0.0.0.0" (:shared-secret auth-tkt-config)
-            {:digest-algorithm (:digest-algorithm auth-tkt-config)
-             :timestamp (java.util.Date.)
-             :user-id (:id login-info)
-             :user-data ""})
-          (:domain auth-tkt-config))
+        (let [current-zoned-timestamp (java-time.zone/with-zone-same-instant (java-time.zone/zoned-date-time) "UTC")]
+          (with-auth-tkt
+            (http/transit-response
+              {:success? true
+               :session-data
+               (let [user (users/find-user db (:id login-info))]
+                 (transport-operator/get-user-transport-operators-with-services db (:groups user) (:user user)))}
+              200)
+            (cookie/unparse "0.0.0.0" (:shared-secret auth-tkt-config)
+                            {:digest-algorithm (:digest-algorithm auth-tkt-config)
+                             :timestamp        (java.util.Date.)
+                             :user-id          (:id login-info)
+                             :user-data        {:session-start-timestamp current-zoned-timestamp}})
+            (:domain auth-tkt-config)
+            ; Define the absolute session timeout in hours.
+            ; If you want to use minutes or seconds instead of hours,
+            ; change the line below to match the unit in use.
+            (.plus current-zoned-timestamp (java-time.amount/hours 8))
+            false))
 
         (http/transit-response {:error :unconfirmed-email} 401)) ;; This could be 403 instead
       ;; Login shall indicate error in a general way and not reveal which credential was invalid
       (http/transit-response {:error :login-error} 400))
     (http/transit-response {:error :login-error} 400)))
 
 (defn- logout [auth-tkt-config]
-  (with-auth-tkt (http/transit-response :ok) "" (:domain auth-tkt-config)))
+  (with-auth-tkt (http/transit-response :ok) "" (:domain auth-tkt-config) (unix-epoch) true))
```

Comment:
Remove parenthesis around `unix-epoch` since it's a value now :)

Reply:
Fixed :)